modules/ztp-image-based-upgrade-concept.adoc

```diff
@@ -0,0 +1,134 @@
+The stage transitions are different from the `ImageBasedUpgrade` API as the `ImageBasedGroupUpgrade` API allows you to combine several stage transitions, also called actions, into one step that share one rollout strategy.
+
+[id="ztp-image-based-upgrade-supported-combinations_{context}"]
+== Supported action combinations
+
+Actions are the list of stage transitions that {cgu-operator} completes in the steps of an upgrade plan for the selected group of clusters.
+Each `plan` entry in the `ImageBasedGroupUpgrade` CR is a separate step and a step contains one or several actions that share the same rollout strategy.
+You can achieve more control over the rollout strategy for each action by separating actions into steps.
+
+These actions can be combined differently in your upgrade plan and you can append subsequent steps later.
+However, you must assess the overall result of the previous steps before appending a step to your plan.
+Either the `Abort` or `Rollback` action must be the first action of a separate step that you append for a set of clusters that failed a previous step.
+
+[IMPORTANT]
+====
+However, you cannot remove actions or steps from an ongoing plan.
+====
+
+The following table shows example plans for different levels of control over the rollout strategy:
+
+.Example upgrade plans
+[cols=3*, width="100%", options="header"]
+|====
+|All actions share same strategy
+|Some actions share same strategy
+|All action share same strategy
+
+a|[source,yaml]
+----
+plan:
+- actions: ["Prep", "Upgrade", "FinalizeUpgrade"]
+    rolloutStrategy:
+      maxConcurrency: 200
+      timeout: 60
+----
+a|[source,yaml]
+----
+plan:
+- actions: ["Prep", "Upgrade"]
+    rolloutStrategy:
+      maxConcurrency: 200
+      timeout: 60
+- actions: ["FinalizeUpgrade"]
+    rolloutStrategy:
+      maxConcurrency: 500
+      timeout: 10
+----
+a|[source,yaml]
+----
+plan:
+- actions: ["Prep"]
+    rolloutStrategy:
+      maxConcurrency: 200
+      timeout: 60
+- actions: ["Upgrade"]
+    rolloutStrategy:
+      maxConcurrency: 200
+      timeout: 20
+- actions: ["FinalizeUpgrade"]
+    rolloutStrategy:
+      maxConcurrency: 500
+      timeout: 10
+----
+|====
+
+[IMPORTANT]
+====
+The clusters that fail one of the actions will skip the rest of the actions in the same step.
+====
+
+The {lcao} accepts the following actions:
+
+`Prep`:: Move to the `Prep` stage.
+`Upgrade`:: Move to the `Upgrade` stage.
+`FinalizeUpgrade`:: Move to the `Idle` stage by finalizing the upgrade on selected clusters that completed the `Upgrade` action.
+`Rollback`:: Move to the `Rollback` stage on all selected clusters.
+`FinalizeRollback`:: Move to the `Idle` stage to finalize a rollback.
+`AbortOnFailure`:: Move to the `Idle` stage to cancel the upgrade on selected clusters that failed the `Prep` or `Upgrade` actions.
+`Abort`:: Move to the `Idle` stage to cancel the upgrade on all selected clusters.
```

Comment:
not upgraded clusters only

Reply:
WDYT?
> Cancel an ongoing upgrade only on clusters that are not upgraded yet by moving to the `Idle` stage.

Please note that I also rephrased these descriptions to put emphasis on **what** the action does instead of **how** (Cancel by moving back to Idle, start upgrade by moving to Upgrade, etc).